coalib/output/ConsoleInteraction.py

```diff
@@ -599,12 +614,22 @@ def print_actions(console_printer, section, actions, failed_actions):
                             values for the action. If the user did
                             choose to do nothing, return (None, None).
     """
-    choice = choose_action(console_printer, actions)
 
-    if choice == 0:
-        return None, None
-
-    return get_action_info(section, actions[choice - 1], failed_actions)
+    reprompt = section.get('reprompt').value
+    if section.name == 'cli':
+        if (not reprompt or StringConverter(reprompt).__bool__()):
+            choice = choose_action(console_printer, actions)
+            if choice == 0:
+                return None, None
+            return get_action_info(section, actions[choice - 1],
+                                   failed_actions)
+        else:
+            return None, None
+    else:
+        choice = choose_action(console_printer, actions)
+        if choice == 0:
+            return None, None
+        return get_action_info(section, actions[choice - 1], failed_actions)
```

Comment:
the way i see it this could be refactored as this:
https://pastebin.com/NqYPJz5G

i might be wrong, please double check it, but i think it looks slightly better